sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -925,6 +925,26 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("show table extended ... partition") {
```

Comment:
In `show-tables.sql`, we only output the value of the column `tableName`, we should verify the schema here.